code/controllers/subsystem/jobs.dm

```diff
@@ -643,6 +643,26 @@ SUBSYSTEM_DEF(jobs)
 			oldjobdatum.current_positions--
 			newjobdatum.current_positions++
 
+/datum/controller/subsystem/jobs/proc/notify_dept_head(jobtitle, antext)
+	// Used to notify the department head of jobtitle X that their employee was brigged, demoted or terminated
+	if(!jobtitle || !antext)
+		return
+	var/datum/job/tgt_job = GetJob(jobtitle)
+	if(!tgt_job)
+		return
+	if(!tgt_job.department_head[1])
+		return
+	var/boss_title = tgt_job.department_head[1]
+	var/obj/item/pda/target_pda
+	for(var/obj/item/pda/check_pda in GLOB.PDAs)
+		if(check_pda.ownrank == boss_title)
+			target_pda = check_pda
```

Comment:
best to break after this. Behavior might change a bit but it's arbitrary anyway. Will save a bit of performance

Reply:
The reason I did not was that there are some cases where a job might have multiple bosses, and both deserve to be notified. E.g: genetics. If I did insert a break here, it would lead to bugs where notifications wouldn't be reliable for jobs like that.

Reply:
Only one of them will be notified here. Either the RD or the CMO. Depending who is first in the `department_head` list